openshift/deploy.sh

```diff
@@ -28,4 +127,9 @@ oc_process_apply ${templates_dir}/worker.yaml "-p WORKER_ADMINISTRATION_REGION=a
 oc_process_apply ${templates_dir}/worker.yaml "-p WORKER_ADMINISTRATION_REGION=api -p WORKER_INCLUDE_QUEUES=GraphImporterTask -p WORKER_NAME_SUFFIX=-graph-import"
 oc_process_apply ${templates_dir}/server.yaml
 oc_process_apply ${templates_dir}/jobs.yaml
-
+oc_process_apply ${templates_dir}/scaler.yaml "-p DC_NAME=bayesian-worker-ingestion -p SQS_QUEUE_NAME=ingestion_bayesianFlow_v0 -p MAX_REPLICAS=8 -p DEFAULT_REPLICAS=4"
```

Comment:
I'd lower the default number of replicas for the dev cluster. Let's go with 2, WDYT?